src/chromium/crConnection.ts

```diff
@@ -62,15 +58,15 @@ export class CRConnection extends EventEmitter {
     const message: ProtocolRequest = { id, method, params };
     if (sessionId)
       message.sessionId = sessionId;
-    if (this._logger.isEnabled())
-      this._logger.info('SEND ► ' + rewriteInjectedScriptEvaluationLog(message));
+    if (debugLogger.isEnabled('protocol'))
```

Comment:
We are resetting the present assumptions on how we log on backend, right? I.e. protocol will just be stored in the server-side log. :+1:

Reply:
Yeah, we are rethinking the logger with rpc. Currently, api calls are sent to the LoggerSink, the rest is available under DEBUG environment variable.